Source/Urho3D/CSharp/Swig/generated/Urho3D/_pre_resource.i

```diff
@@ -28,7 +28,7 @@
 %csattribute(Urho3D::Resource, %arg(unsigned int), MemoryUse, GetMemoryUse, SetMemoryUse);
 %csattribute(Urho3D::Resource, %arg(unsigned int), UseTimer, GetUseTimer);
 %csattribute(Urho3D::Resource, %arg(Urho3D::AsyncLoadState), AsyncLoadState, GetAsyncLoadState, SetAsyncLoadState);
-%csattribute(Urho3D::Resource, %arg(ea::string), NativeFileName, GetNativeFileName);
+%csattribute(Urho3D::Resource, %arg(ea::string), NativeFileName, GetAbsoluteFileName);
```

Comment:
Should be `%csattribute(Urho3D::Resource, %arg(ea::string), AbsoluteFileName, GetAbsoluteFileName);`. Next time script generates these statements it would change anyhow.